README.md

```diff
@@ -1 +1,74 @@
-Keep your branches clean and nice. Use flake8 and tests. Good luck!
\ No newline at end of file
+## Project description
+This web application allows you to create a local and public IP entry, view and email a list of data
+
+## Quick start
+### First step: 
+- Copy project from git
+```
+git clone https://github.com/nester256/rpm-hws_7-1_2023
+
+cd rpm-hws_7-1_2023
+
+git checkout nesterov
+```
+### Second step:
+- Create docker image 
+```
+docker run  -d \
+        --name ips_test \
+        -e POSTGRES_USER=admin \
+        -e POSTGRES_PASSWORD=1234 \
+        -e PGDATA=/postgres_data_inside_container \
+        -v ~/sirius_db_2023/postgres_data:/postgres_data_inside_container \
+        -p 38746:5432 \
+        postgres:15.1
+```
+### Third step:
+- Create tables
+```
+psql -h 127.0.0.1 -p 5435 -U admin -d ips_tests -f db_init.ddl
```

Comment:
is it so hard for u to check your own commands?

you have set up a docker container with port 38746, now suggest connecting to 5435

Reply:
`-d ips_tests` .... :disappointed: 
-d is supposed to be a dbname, not a docker container name.. it wont work anyway

Reply:
I will repeat once again that I have never used this command and am not going to, so I don’t understand how to write it correctly